src/main/resources/languages/en/messages.yml

```diff
@@ -230,7 +231,7 @@ messages:
   fire-extinguish: '&7You have extinguished yourself'
   cannot-place: '&cYou cannot place that block there!'
   no-pvp: '&cYou cannot pvp in here!'
-  radiation: '&4You have been exposed to deadly radiation! &cGet rid of the radioactive item or equip the complete hazmat suit!'
+  radiation-exposed: '&4You have been exposed to radiation! &cGet rid of the radioactive item or equip the complete hazmat suit!'
```

Comment:
I don't think we should change the key for this, it will break every language despite the message not changing

Reply:
I'd recommend the removal of the deadly part at least.